src/app/datagrid/selection-single/selection-single.html

```diff
@@ -137,7 +145,11 @@ <h3>Server-driven, trackBy item</h3>
   <clr-dg-column>Creation date</clr-dg-column>
   <clr-dg-column>Favorite color</clr-dg-column>
 
-  <clr-dg-row *ngFor="let user of trackByIdServerUsers; trackBy: trackById" [clrDgItem]="user">
+  <clr-dg-row
+    *ngFor="let user of trackByIdServerUsers; trackBy: trackById"
+    [clrDgItem]="user"
+    [clrDgRowAriaLabel]="'Select row for ' + user.name"
```

Comment:
This updates the demos on the angular docs application to create unique user names.